nostrpostrlib/src/test/java/nostr/postr/UtilsTest.kt

```diff
@@ -37,4 +37,19 @@ internal class UtilsTest {
         assertArrayEquals(sharedSecretExpected, sharedSecretAliceToBob)
         assertArrayEquals(sharedSecretExpected, sharedSecretBobToAlice)
     }
+
+    @Test
+    fun convertPubkeyHexToNpub(){
+        val pubkeyHex = "e4c47aedea8ea54255f5ba07a77053b24553e9b975435e56da343da19aec7881"
+        val npubRepresentation = Hex.decode(pubkeyHex).toNpub()
+        val correctRep = "npub1unz84m0236j5y404hgr6wuznkfz486dew4p4u4k6xs76rxhv0zqsq9q6t8"
+
+        val jackPubkeyHex = "82341f882b6eabcd2ba7f1ef90aad961cf074af15b9ef44a09f9d2a8fbfbe6a2"
+        val derivedJackNpub = Hex.decode(jackPubkeyHex).toNpub()
+        val correctJackNpub = "npub1sg6plzptd64u62a878hep2kev88swjh3tw00gjsfl8f237lmu63q0uf63m"
+
+        assertEquals(correctRep, npubRepresentation)
+        assertEquals(correctJackNpub, derivedJackNpub)
```

Comment:
Would you mind sharing where the test vectors are from? And privkey would also be nice to have.

Reply:
OK No problem. The first one, I got from a person who converted their hex-formatted key to bech32. And the second is Jack Dorsey's pubkey.

Reply:
Can you add that as a code comment? Link to the tool used to convert for example.